### ISSUE TITLE
minified files not in .gitignore

### ISSUE THREAD

**Issue by Justinvolved** [dynamicweb/Swift#20]
Since all the .min.css and and .min.js files are generated during build:wenpack would it not be better to ignore these and not have the compiled versions in Git?

**Comment by misterwinther** [comment]
Yes @Justinvolved , for sure. Also to clean them from the repo. We have an idea to refactor the frontend setup. New structure, scripts, build/compile tooling etc. in the next couple of sprints. So until then... boo hoo 😄  

//Stefan
